packages/core/src/modules/ledger/services/LedgerService.ts

```diff
@@ -97,17 +93,23 @@ export class LedgerService {
         this.logger.debug(`Pool '${poolName}' already exists`, {
           indyError: 'PoolLedgerConfigAlreadyExistsError',
         })
+      } else if (isIndyError(error)) {
+        throw new IndySdkError(error)
```

Comment:
```suggestion
        throw isIndyError(error) ? new IndySdkError(error) : error
```

Could be a ternary expression